content/applications/finance/fiscal_localizations/peru.rst

```diff
@@ -588,3 +588,265 @@ As part of the Peruvian localization, besides creating credit notes from an exis
+   
+If that is not the case, be sure to fulfill the rest of the information 
+needed for the vehicle.
+
+.. image:: peru/vehicle_not_M1_or_L_pe.png
+   :align: center 
+   :alt: Vehicle not selected as an M1 or L type with extra fields shown
+   
+Product Configuration
+*********************
+
+Make sure that the applicable information in the product form is fully 
+configured. The *“Partida Arancelaria” (Tariff Item)* the field needs to be filled. 
+
+To do this, go to :menuselection:`Inventory App --> Products` open the 
+product to be configured and fill the “Partida Arancelaria” field.
+
+Common Errors
+*************
+
+- `Diferente prefijo para productos (T001 en algunos, T002 en otros)`: At the moment,
+Odoo does not support the automation of prefixes for products. This can be done 
+manually for each product output. This can be done as well for non-storable products, 
+however keep in mind that there will be no traceability.
+
+- `2325 - GrossWeightMeasure - El dato no cumple con el formato establecido "Hace 
+falta el campo" "Peso"" en el producto`: This error occurs when the weight on the 
+product is set as “0.00”. To fix, you will need to cancel the Waybill and recreate 
+it. Make sure that you fix the weight on the product before creating the new 
+Waybill, or it will throw you the same error.
+
+- `JSONDecodeError: Expecting value: line 1 column 1 (char 0) when creating a Delivery 
+Guide`: This error is typically generated due to SOL user issues. Verify the user’s 
+connection with the SUNAT; the SOL user must be established with the company RUT + user ID.
+Example: 2012188549JOHNSMITH
+
+- `El número de documento relacionado al traslado de mercancía no cumple con el formato 
+establecido : error: documento relacionado: The “Related Document Type” and “Related Document Number”
+fields only apply to Invoices and Receipts.
+
+- `400 Client error: Bad Request for URL`: This error is not solvable on Odoo’s side; 
+it is advised you reach out to the SUNAT and verify the user. It may be necessary to 
+create a new user.
+
+- `Invalid content was found starting with element 'cac:BuyerCustomerParty'`: This 
+error occurs when the transfer reason is set as “other”. Please select another option. 
+According to the official documentation of the SUNAT’s Waybill Guide, the transfer
+reason 03 (sale with shipment to third party) or 12 (others) will not work correctly
+in Odoo, since you should not put an “empty” or “blank” customer.
+
+- `Duda cliente: consumo de créditos IAP al usar GRE 2.0`: For live clients using 
+IAP: No credit is consumed (in theory) because it does not go through the OSE, i.e.
+these documents are directly sent to the SUNAT.
+
+- `Errores con formato credenciales GRE 2.0 (traceback error)`: Odoo currently throws
+an error with a traceback, instead of a message that the credentials are not correctly
+configured in the database. If this occurs on your database, please verify your credentials.
```

Comment:
```suggestion
Common errors
~~~~~~~~~~~~~

- `Diferente prefijo para productos (T001 en algunos, T002 en otros)`

  At the moment, Odoo does not support the automation of prefixes for products. This can be done
  manually for each product output. This can be done as well for non-storable products, however
  keep in mind that there will be no traceability.
- `2325 - GrossWeightMeasure - El dato no cumple con el formato establecido "Hace falta el campo"
  "Peso"" en el producto`

  This error occurs when the weight on the product is set as `0.00`. To fix this, you will need to
  cancel the waybill and recreate it. Make sure that you fix the weight on the product before
  creating the new waybill, or it will result in the same error.
- `JSONDecodeError: Expecting value: line 1 column 1 (char 0) when creating a Delivery Guide`

  This error is typically generated due to SOL user issues. Verify the user's connection with the
  SUNAT; the SOL user must be established with the company RUT + user ID. For example:
  `2012188549JOHNSMITH`.
- `El número de documento relacionado al traslado de mercancía no cumple con el formato establecido
  : error: documento relacionado`

  The *Related Document Type* and *Related Document Number* fields only apply to invoices and
  receipts.
- `400 Client error: Bad Request for URL`

  This error is not solvable from Odoo; it is advised you reach out to the SUNAT and verify the
  user. It may be necessary to create a new user.

- `Invalid content was found starting with element 'cac:BuyerCustomerParty'`

  This error occurs when the transfer reason is set as *other*. Please select another option.
  According to the official documentation of the SUNAT's waybill guide, the transfer reason *03
  (sale with shipment to third party)* or *12 (others)* will not work correctly in Odoo, since you
  should not have an empty or blank customer.
- `Duda cliente: consumo de créditos IAP al usar GRE 2.0`

  For live clients using IAP, no credit is consumed (in theory) because it does not go through the
  OSE, i.e. these documents are directly sent to the SUNAT.
- `Errores con formato credenciales GRE 2.0 (traceback error)`

  Odoo currently throws an error with a traceback, instead of a message that the credentials are not
  correctly configured in the database. If this occurs on your database, please verify your
  credentials.s occurs on your database, please verify your credentials.
```